cli/azd/test/functional/env_test.go

```diff
@@ -230,16 +274,58 @@ func envSetValue(ctx context.Context, t *testing.T, cli *azdcli.CLI, key string,
 	require.NoError(t, err)
 }
 
-func envGetValues(ctx context.Context, t *testing.T, cli *azdcli.CLI, args ...string) map[string]string {
-	defaultArgs := []string{"env", "get-values", "--output", "json"}
+func envGetValues(
+	ctx context.Context,
+	t *testing.T,
+	cli *azdcli.CLI,
+	args ...string) map[string]string {
+	defaultArgs := []string{"env", "get-values"}
 	args = append(defaultArgs, args...)
 
 	result, err := cli.RunCommand(ctx, args...)
 	require.NoError(t, err)
 
-	var envValues map[string]string
-	err = json.Unmarshal([]byte(result.Stdout), &envValues)
-	require.NoError(t, err)
+	outputMode := azdcli.GetOutputFlagValue(args)
+
+	envValues := map[string]string{}
+
+	switch outputMode {
+	case "json":
+		err = json.Unmarshal([]byte(result.Stdout), &envValues)
+		require.NoError(t, err)
+	case "", "none":
+		scanner := bufio.NewScanner(strings.NewReader(result.Stdout))
+		for scanner.Scan() {
+			line := scanner.Text()
+			if !strings.Contains(line, "=") {
+				t.Fatal("unexpected line in output: " + line)
+			}
+
+			parts := strings.Split(line, "=")
```

Comment:
This doesn't strictly validate for an unevaluated `.env` line, but I'm also not sure what we contractually signed up for.